src/components/Post.js

```diff
@@ -53,16 +55,16 @@ class Post extends React.Component {
   }
 
   render () {
-    const { post, communities, comments, expanded, onExpand } = this.props
+    const { post, communities, comments, expanded, onExpand, community } = this.props
     const { tag, media } = post
     const image = find(media, m => m.type === 'image')
     const classes = cx('post', tag, {image, expanded})
-    const title = decode(post.name || '')
+    const title = formatPostTitle(decode(post.name || ''), (community || {}).slug)
```

Comment:
i recall that we intentionally decided not to make urls in titles into clickable links -- this changes that. @edwardwest can you confirm?